blockchain/chain/src/store/chain_store.rs

```diff
@@ -613,6 +656,32 @@ where
     Ok(out)
 }
 
+#[cfg(feature = "json")]
+pub mod headchange_json {
+    use super::*;
+    use blocks::tipset_json::TipsetJson;
+    use serde::{Deserialize, Serialize};
+
+    #[derive(Debug, Serialize, Deserialize)]
+    #[serde(untagged)]
+    #[serde(rename_all = "PascalCase")]
```

Comment:
Is this untagged or tagged with PascalCase? These are contradictory. How are you checking the format?

Reply:
This was a mistake, it is supposed to be tagged

Reply:
But the enum variants are not pascal cased in Lotus? They are all lower case from what I can see